README.md

```diff
@@ -1,2 +1,33 @@
 # Apple-website-webpage-clone
 This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.
+
+
+[screenshot](screenshot.PNG)
+
+## Live Demo
+---
+[Apple-website-webpage-clone] https://ecstatic-lamarr-cd95c7.netlify.app/
```

Comment:
- [ ] Please fix the improper applied and broken links in your README to make your project look more professional.